docs/user-guide/configuration/externalConfig.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: main
+title: External Config
+category: User Guide
+menu: menu
+toc:
+    - title: External Config
+      url: "#external-config"
+
+---
+# External Config
+External config allows a single parent pipeline to create and manage a list of child pipelines with job configuration from parent's `screwdriver.yaml` and source code from itself.
+
+This feature will benefit users who owning multiple pipelines with same building patterns to manage all pipelines easily.
+
+## Configure external config in parent pipeline
+In your parent repository's `screwdriver.yaml`, you can define child pipelines with syntax `childPipelines`. We will create child pipelines with the specified scmUrls for you. Please make sure you have **admin** access towards child repositories otherwise we won't create the child pipeline.
```

Comment:
In your parent repository's `screwdriver.yaml`, you can define child pipelines with the keyword `childPipelines`. Screwdriver will create or delete child pipelines on our behalf based on the listed `scmUrls`. You must have **admin** access in each child repository in order to manage child pipelines through this feature.

Reply:
Rather than calling child pipelines, give the meaning of child/external source repositories ? I feel rather than pipeline we need to give the message of one single pipeline config for building multiple child/external source repositories.

Reply:
Rather than calling child pipelines, give the meaning of child/external source repositories ? I feel rather than pipeline we need to give the message of one single pipeline config for building multiple child/external source repositories ?